test/functional/fixtures/driver/testcafe-fixtures/clear-and-lock-storages.js

```diff
@@ -0,0 +1,32 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `Storages`
+    .page `http://localhost:3000/fixtures/driver/pages/set-value-to-storages.html`;
+
+const saveNativeStorageKey = ClientFunction(storageName => {
```

Comment:
Let's call if `setNativeStorageKey `.
And we can do not return value from this method